blocks/search-results/search-results.js

```diff
@@ -0,0 +1,42 @@
+import { createCardBlock } from '../card/card.js';
+import { loadBlock } from '../../scripts/lib-franklin.js';
+
+export async function searchResults(params, jsonData) {
+  // eslint-disable-next-line
+    return jsonData.data.filter((entry) => (entry.title + entry.content + entry.path + entry.authors + entry.collections + entry.section + entry.categories + entry.template).toLowerCase().includes(params.toLowerCase()));
+}
+
+export default async function decorate(block) {
+  // fetch results from json files
+  block.innerHTML = '';
+  console.log(window.location.origin);
+  const allData = await fetch(`${window.location.origin}/articles.json?sheet=full`);
```

Comment:
This currently blocks rendering until the results are ready. Instead move the code for fetching and updating the dom into an async method. Then do NOT `await` that method call. Until the results are ready, show the h1 and some simpler spinner. see https://github.com/hlxsites/zemax/pull/178/files#diff-c47712f82ae3bdf9f1324059441a3ad1fae641c4b1fb97724dfa2d87e4818484R28-R42

This should fix the LHS.

Reply:
Done. With few doubts though.